packages/webapp/src/components/Crop/ManagementPlanName/index.jsx

```diff
@@ -47,9 +49,13 @@ export default function PureManagementPlanName({
     shouldUnregister: false,
     user: user,
     users: users,
-    defaultAssignee: { label: t('TASK.UNASSIGNED'), value: null, isDisabled: false },
-    disableUnAssignedOption: false,
+    isAssigned: false,
+    defaultAssignee: unassigned,
     additionalFields: {
+      [ASSIGNEE]:
+        users.length === 1
+          ? { label: user.first_name, value: user.user_id, isDisabled: false }
+          : unassigned,
```

Comment:
Can we have these as `defaultAssignee`(line 53)? I think it will be the same thing...
and user's `last_name` is missing: `${user.first_name} ${user.last_name}`

Reply:
Added. Yes, don't think that was necessary.